sql/core/benchmarks/FilterPushdownBenchmark-results.txt

```diff
@@ -0,0 +1,556 @@
+############################[ Pushdown for many distinct value case ]############################
```

Comment:
@wangyum, can we mimic any other format? For example, when I do such thing, I usually copy a format from another. For example, how about those below (which I am kind of used to).

```
Pushdown for many distinct value case:

  Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
  Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz

  Select 0 string row (value IS NULL):     Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
  ------------------------------------------------------------------------------------------------
  Parquet Vectorized                            7928 / 8019          2.0         504.0       1.0X
```

```
========================================================================
Pushdown for many distinct value case
========================================================================

Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz

Select 0 string row (value IS NULL):     Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
Parquet Vectorized                            7928 / 8019          2.0         504.0       1.0X
```

```
Pushdown for many distinct value case ...

Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz

Select 0 string row (value IS NULL):     Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
Parquet Vectorized                            7928 / 8019          2.0         504.0       1.0X
```

Reply:
How about this?
```
...
Select all int rows (value != -1):       Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
Parquet Vectorized                            1140 / 1165          0.9        1087.4       1.0X
Parquet Vectorized (Pushdown)                 1140 / 1172          0.9        1086.8       1.0X
Native ORC Vectorized                         1158 / 1206          0.9        1104.7       1.0X
Native ORC Vectorized (Pushdown)              1151 / 1220          0.9        1098.1       1.0X


================================================================================================
Pushdown for few distinct value case (use dictionary encoding)
================================================================================================

Java HotSpot(TM) 64-Bit Server VM 1.8.0_151-b12 on Mac OS X 10.12.6
Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz

Select 0 distinct string row (value IS NULL): Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
------------------------------------------------------------------------------------------------
Parquet Vectorized                             512 /  565          2.0         488.6       1.0X
Parquet Vectorized (Pushdown)                   27 /   33         39.3          25.5      19.2X
Native ORC Vectorized                          509 /  546          2.1         485.0       1.0X
Native ORC Vectorized (Pushdown)                79 /   91         13.2          75.5       6.5X
...
```

Reply:
Yup, to be looks better. Let's go this way and correct it if any other committers have other preferences.

Reply:
BTW that format was from our testing script as you might already know :-).

Reply:
OK, I rebuild the benchmark use this format.